### DISCUSSION TITLE
Unable to push to github using  correct username and password

### DISCUSSION THREAD

**Discussion by shibasundarbehera** [community/community#58808] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Unable to push to github using  correct username and password  

remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.

any clue ?

**Comment by airtower-luna** [comment, marked as answer]
As the message says, you cannot use your password for `git push` or other Git operations on GitHub (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Reply by Anusha-296** [reply]
I am using token for password but also error

**Comment by shibasundarbehera** [comment]
Thank you @airtower-luna  . It worked !!.
